news/_posts/2021-07-06-responsivebo.md

```diff
@@ -0,0 +1,132 @@
+--- 
+layout: post
+title:  "Towards a more mobile-friendly back office"
+subtitle: "Quick wins of the responsive back office"
```

Comment:
This sentence sounds a bit weird to me. What do you think of "Quick wins for a responsive back office"?